Client/Frontend/Browser/MetadataParserHelper.swift

```diff
@@ -48,13 +48,5 @@ class MetadataParserHelper: TabEventHandler {
             ]
             NotificationCenter.default.post(name: .OnPageMetadataFetched, object: nil, userInfo: userInfo)
         }
-        
-        // Ecosia: read local storage value for cookie consent
-        if url.normalizedHost?.hasPrefix("ecosia.org") == true, !tab.isPrivate {
-            webView.evaluateJavascriptInDefaultContentWorld("localStorage.getItem(\"\(Cookie.consentKey)\")") { result, error in
```

Comment:
Haven't checked with @ecotopian on other reasons for storing that via the `localStorage` via this particular logic placed here and from what the ticket states, a modernisation was expected to be tackled as part of the Cookie store rework.
Let's see if any, in case revert it, otherwise leave it as is 💪

Reply:
We chatted async on that. We decided to proceed and leave the Cookie implementation in the centralized `Core/Cookie.swift` file, thus leveraging maintainability in the longer term. 
If anything comes up out of this implementation, we'll tackle it as a different piece of work.